usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -253,7 +278,7 @@ This endpoint returns a list of data that is associated with the award profile p
 + `city_name`: `McLean` (required, string, nullable)
 + `county_name`: `Fairfax` (required, string, nullable)
 + `state_code`: `VA` (required, string, nullable)
-+ `state_name`: `Virginia` (required, string, nullable)
++ `state_name`: `Virginia` (optional, string, nullable)
```

Comment:
This isn't necessary due to #2105

Reply:
yeah i had changed it just to check if the dredd test were passing, it should be goo now